plugins/org.eclipse.fordiac.ide.typemanagement/src/org/eclipse/fordiac/ide/typemanagement/navigator/ToolLibraryLabelProvider.java

```diff
@@ -46,11 +46,11 @@ public void removeListener(final ILabelProviderListener listener) {
 	public Image getImage(final Object element) {
 		if (element instanceof IProject) {
 			return FordiacImage.ICON_TYPE_NAVIGATOR.getImage();
-		} else if (element instanceof IFolder && ((IFolder) element).isLinked()) {
+		}
+		if (element instanceof final IFolder folder && folder.isLinked()) {
```

Comment:
Could just merge the if clause here with `||` as they return the same value

Reply:
Yes that is a good point. Fixed.